__tests__/frontend-proxy.ts

```diff
@@ -96,26 +96,40 @@ test('removes trailing slashes and prepends language code when the backend is fr
 })
 
 describe('when request contains content api parameter', () => {
-  let response: Response
-
-  beforeEach(async () => {
+  beforeEach(() => {
     setupProbabilityFor(Backend.Frontend)
 
-    response = await currentTestEnvironment().fetch({
-      subdomain: 'en',
-      pathname: '/?contentOnly',
+    givenUuid({
+      id: 1555,
+      __typename: 'Article',
+      alias: '/1555',
+      instance: Instance.En,
     })
   })
 
-  test('chooses legacy backend', async () => {
-    await expectLegacy(response)
-  })
+  test('chooses frontend', async () => {
+    const response = await currentTestEnvironment().fetch({
+      subdomain: 'en',
+      pathname: '/1555?contentOnly',
+    })
 
-  test('does not set cookie with random number', () => {
-    expect(response.headers.get('Set-Cookie')).not.toEqual(
-      expect.stringContaining('useFrontend')
+    expect(response.status).toBe(200)
+    expect(await response.text()).toEqual(
+      expect.stringContaining('(no header)')
     )
   })
+
+  // TODO: return 301. could not find the reason so far
```

Comment:
@kulla do you know why this might happen?

Reply:
That's fine: `/1555/?contentOnly` gets redirected to `/1555?contentOnly` by our redirect module...